src/components/BuildingApp/BuildingsForm/index.js

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import Button from '../../sharedComponents/Button';
+import styles from './BuildingsForm.module.css';
+
+const BuildingsForm = ({
+  onSubmit,
+  onClose,
+  building,
+}) => {
+  const [state, setState] = useState({
+    name: building.name || 'Building name..',
```

Comment:
The default values for strings should be empty strings:
```suggestion
    name: building.name || '',
```

You can add these strings as placeholders